exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md

```diff
@@ -0,0 +1,11 @@
+
+The AMR-Wind computational domain extends from -10m to 10m in the x direction, which is only a portion of the length of the original wave tank, while it extends from -2m to 2m in y, the full width of the tank, and from -3.6m to 2.4m in z, reaching the full depth of the tank and providing enough room around the vertical extent of the platform. Two levels of refinement are added, and these are targeted at the position of the air-water interface and the platform geometry. The refinements reach a resolution of 31.25mm in the x and y directions and 15.625mm in the z direction. The inlet in x uses a wave generation boundary condition, and the waves are forced into the domain with a relaxation zone. The outlet in x has a pressure outflow condition and the waves are absorbed using a numerical beach. The other boundaries are represented as slip walls.
+
+The Nalu-Wind computational mesh is tetrahedral and adheres to the mesh resolution recommendations in section 3.2.3 of this paper: https://www.sciencedirect.com/science/article/pii/S0029801822013580. (Need way to access mesh)
+
+The irregular wave field forced into the AMR-Wind domain is first generated by the high-order spectral solver HOS-NWT, which produces a file containing a time series of wave modes. These are read into AMR-Wind, transformed, and interpolated onto the computational mesh during the ExaWind simulation. When selecting the wave setup for HOS-NWT, a variety of approaches were evaluated, including iterative approaches and multi-wave decompositions, to reach similarity with the observed spectrum from the MARIN experiment. Ultimately, the most similar wave case was produced by using the built-in capability of HOS-NWT to produce waves using the JONSWAP spectrum with $\gamma = 3.3$, as listed in the OC6 case 3.3 description. These waves have a significant wave height of 0.148m and a peak period of 1.6971s. In HOS-NWT, the wave simulation was performed in two dimensions (2D), eliminating the spanwise direction, which is incorporated only in the context of the ExaWind simulation. (Need way to access modes file)
+
+This case is currently being run to reach a minimum duration of 9 minutes, but here is a low-frame-rate video from the first minute, showing the evolution of the air-water interface within the Nalu-Wind portion of the domain.
+
+![OC6 fixed platform early](/Users/mkuhn/exawind-benchmarks/exawind/OC6_platform_wave_tank/floater_interface_nalu.mp4)
```

Comment:
This location of the video looks like it's relative to a location on your local machine.  I think you can do a link to an mp4 video in markdown in github and have it show up, but not sure if it'll appear on a sphinx webpage.

Reply:
good catch. I'm going to try with a relative link, then hopefully when we port it to sphinx we can copy the video file to the same relative location.